pkg/fetch/bucket.go

```diff
@@ -41,7 +42,7 @@ func (f *bucketFetcher) File(pathElements ...string) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	content, err := f.bucket.ReadAll(context.TODO(), rp)
+	content, err := f.bucket.ReadAll(f.context, rp)
```

Comment:
I think it would be better if `ctx` was the first arg of the `File` func, but out of scope for this PR I guess.